server/src/sdk/browserSide/pageAnalyzer.js

```
@@ -0,0 +1,2558 @@
+                    counter--;
+                  }
+                }
+                value = '\\' + codePoint.toString(16).toUpperCase() + ' ';
+              } else {
+                if (options.escapeEverything) {
+                  if (regexAnySingleEscape.test(character)) {
+                    value = '\\' + character;
+                  } else {
+                    value = '\\' + codePoint.toString(16).toUpperCase() + ' ';
+                  }
+                } else if (/[\t\n\f\r\x0B]/.test(character)) {
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)